client/src/config.rs

```diff
@@ -0,0 +1,23 @@
+use serde::Deserialize;
+use std::net::SocketAddr;
+use std::str::FromStr;
+
+pub(crate) const DEFAULT_NODE_ADDR: &str = "127.0.0.1:3030";
+pub(crate) const DEFAULT_PROTOCOL: &str = "http";
+pub(crate) const DEFAULT_HEALTH_ENDPOINT: &str = "health";
```

Comment:
We shouldn't keep command-related constants here

Reply:
The are configs for the node url, the defualt settings, not for the command line